packages/browser/__tests__/login/oidc/IssuerConfigFetcher.spec.ts

```diff
@@ -151,19 +154,4 @@ describe("IssuerConfigFetcher", () => {
       "[https://some.url] has an invalid configuration: Some error"
     );
   });
-
-  it("should store the config under a key specific to the config source", async () => {
```

Comment:
Why is test removed ?

Reply:
From an offline conversation: refactoring made it redundant with another test from the same file.